packages/ui/src/components/SectionHeader.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="header">
+    <router-link to="/">
+      <div class="back-container">
+        <img class="angle" src="@/assets/angle-left.svg" alt="angle-left" />
+        <p>BACK</p>
+      </div>
+    </router-link>
+    <router-link to="/">
+      <img class="ranch-img" src="@/assets/ranchLogo.svg" alt="Back button" />
+    </router-link>
+    <h3 class="title-section">{{ title }}</h3>
+  </div>
+</template>
+
+<script>
+export default {
```

Comment:
Can you define the script as [here](https://github.com/otherplane/witty-bufficorns/pull/43/files#diff-005e7a62d33c583d3d8235c07b70ba23b20909d743d438a849c223df37c64846R8)?